tests/generators/test_classic.py

```diff
@@ -0,0 +1,8 @@
+import pytest
```

Comment:
Question: do we need to import pytest on each test file? I think not... if that's the case we should probably remove it.

Reply:
Good point